fabric-fluid-v1/src/main/java/net/fabricmc/fabric/api/fluid/v1/math/Fraction.java

```diff
@@ -0,0 +1,45 @@
+package net.fabricmc.fabric.api.fluid.v1.math;
+
+import com.google.common.math.LongMath;
+
+/**
+ * a utility class for displaying drops as readable fractions to users.
+ *
+ * @see Drops#asFraction(long)
+ * @see #toString()
+ */
+public final class Fraction {
+	public final long proper, numerator, denominator;
+
+	public Fraction(long proper, long numerator, long denominator) {
```

Comment:
Is this worth keeping around if we're just gonna be working off `Drops` anyway?

Reply:
it's for the toString and nothing else